token-tracker/src/truth-sources/keep-token.js

```diff
@@ -0,0 +1,215 @@
+/** @typedef { import("../lib/context").Context } Context */
+/** @typedef { import("../lib/ethereum-helper").Address } Address */
+
+import BN from "bn.js"
+import pAll from "p-all"
+
+import { ITruthSource, AddressesResolver } from "./truth-source.js"
+
+import { Contract } from "../lib/contract-helper.js"
+
+import { EthereumHelpers } from "@keep-network/tbtc.js"
+const { callWithRetry } = EthereumHelpers
+
+import { getPastEvents } from "../lib/ethereum-helper.js"
+import { dumpDataToFile } from "../lib/file-helper.js"
+import { logger } from "../lib/winston.js"
+
+import KeepTokenJSON from "@keep-network/keep-core/artifacts/KeepToken.json"
+
+const KEEP_TOKEN_HISTORIC_HOLDERS_DUMP_PATH = "./tmp/keep-token-holders.json"
```

Comment:
there is also a function with `dump` -> `dumpDataToFile`

Reply:
Do you think I should also change the name for this function? Does _dump_ also have bad connotations with data?

Reply:
Actually I don't know if it has a bad connotation, I figured I mention it just for consistency since you changed the `const` name of the path. I'm fine with both names.

Reply:
I think it is fine in the context of the data dump, but I'd like to avoid it when it's about tokens.